src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String lines = readFromFile(fromFileName);
+        String report = getReport(lines);
+        writeReport(toFileName, report);
+    }
+
+    private static String readFromFile(String fromFileName) {
+        String lines;
+        StringBuilder linesBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));) {
+            lines = bufferedReader.readLine();
```

Comment:
```suggestion
            line = bufferedReader.readLine();
```
In one moment here is only one line